teuthology/nuke/__init__.py

```diff
@@ -156,11 +166,12 @@ def openstack_remove_again():
     xargs --no-run-if-empty --max-args 1 -P20 openstack server delete --wait
     true
     """)
-    sh("""
-    openstack volume list --name REMOVE-ME --column ID --format value |
-    xargs --no-run-if-empty --max-args 1 -P20 openstack volume delete
-    true
-    """)
+    volumes = OpenStack().run("volume list -f json --long")
```

Comment:
i assume that `volumes` will the output of

```
openstack --quiet volume list -f json --long
```

so it will be a unicode string, right? can we iterate through a unicode string and invoke the `v.get()` on each of its element? because, i think `v` is a `str` presenting an element in `volumes`.

Reply:
@tchaikov I've updated the code a little bit. It actually works in similar way to OpenStack().list_volumes()

Reply:
yeah. now it does.